interpreters/Wasm.mk

```diff
@@ -21,9 +21,14 @@
 ifeq ($(WASM_BUILD),y)
   ifneq ($(CONFIG_INTERPRETERS_WAMR)$(CONFIG_INTERPRETERS_WAMR3),)
 
-WASM_INITIAL_MEMORY ?= 65536
-STACKSIZE           ?= $(CONFIG_DEFAULT_TASK_STACKSIZE)
-PRIORITY            ?= SCHED_PRIORITY_DEFAULT
+STACKSIZE  ?= $(CONFIG_DEFAULT_TASK_STACKSIZE)
+PRIORITY   ?= SCHED_PRIORITY_DEFAULT
+
+ifneq ($(INITMEMORY),)
+  ifeq ($(shell test $(INITMEMORY) -lt $(STACKSIZE); echo $$?),0)
+    INITMEMORY := $(shell expr $(STACKSIZE) + $(INITMEMORY))
```

Comment:
do you still need this?
if a user specifies INITMEMORY smaller than STACKSIZE, it's likely a mistake. i feel it's more user-friendly to just leave the linker complain.

Reply:
This is a configurable value. The stack size requested by the developer may vary under different configurations. I think it is better  to solve this issue in the general makefile, otherwise this logic will appear in some application

Reply:
sorry, i don't understand your logic.
if a developer specified INITMEMORY < STACKSIZE, i'd assume it was a mistake. do you have counter examples?

Reply:
Some stack size configurations may be written like this:

```
config EXAMPLES_HELLO_STACKSIZE
   int "Hello stack size"
   default 16384 if ARCH_SIM
   default DEFAULT_TASK_STACKSIZE
```

It is common to have different stack size on different platforms

Reply:
> Some stack size configurations may be written like this:
> 
> ```
> config EXAMPLES_HELLO_STACKSIZE
>    int "Hello stack size"
>    default 16384 if ARCH_SIM
>    default DEFAULT_TASK_STACKSIZE
> ```
> 
> It is common to have different stack size on different platforms

yes.
it seems wrong to use platform DEFAULT_TASK_STACKSIZE for wasm in the first place.

Reply:
also, i suspect there is no reasonable way to fix the situation (INITMEMORY < STACKSIZE) automatically.
your solution (INITMEMORY = INITMEMORY + STACKSIZE) seems wrong because INITMEMORY means to include STACKSIZE.
only the user who specified INITMEMORY can know how to fix it.